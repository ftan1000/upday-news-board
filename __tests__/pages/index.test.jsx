import React from 'react';
import {shallow} from 'enzyme';

import Home from '../../pages';
import Header from '../../components/header';
import LoggedInFooter from '../../pages/loggedInFooter';
import Boards from '../../components/boards';
import renderer from 'react-test-renderer';

describe('Home', () => {

  it('renders snapshot', () => {
    const component = renderer.create(<Home />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('contains expected components', () => {
    const wrapper = shallow(<Home/>);
    expect(wrapper.find(Header)).toBeTruthy();
    expect(wrapper.find(Boards)).toBeTruthy();
    expect(wrapper.find(LoggedInFooter)).toBeTruthy();
  });
});
