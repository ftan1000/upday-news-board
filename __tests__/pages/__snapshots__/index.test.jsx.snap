// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders snapshot 1`] = `null`;
